scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatWriter.scala

```diff
@@ -594,18 +595,42 @@ class FormatWriter(formatOps: FormatOps) {
           if (matches == 0 || doubleNewline || !locationIter.hasNext) {
             var column = 0
             val columns = minMatches
+
+            /**
+              * Separator length gap needed for align blocks with different token
+              * lengths by expression names, not tokens itself.
+              *
+              * Without gaps considering:
+              * ```
+              * libraryDependencies ++= Seq(
+              *   "org.scalacheck"  %% "scalacheck" % scalacheckV,
+              *   "io.get-coursier" % "interface"   % "0.0.17"
+              * )
+              * ```
+              *
+              * With gaps considering:
+              * ```
+              * libraryDependencies ++= Seq(
+              *   "org.scalacheck" %% "scalacheck" % scalacheckV,
+              *   "io.get-coursier" % "interface"  % "0.0.17"
+              * )
+              * ```
+              * */
+            val previousSeparatorLengthGaps = Array.fill(block.length)(0)
```

Comment:
is it the same as `new Array[Int](block.length)`?

Reply:
Yes